Torch.Server/Views/Converters/ModToIdConverter.cs

```diff
@@ -32,9 +32,14 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
         {
             //if (targetType != typeof(int))
             //    throw new NotSupportedException("ModToIdConverter can only convert mods into int values or vise versa!");
-            var mod = (ModItemInfo) values[0];
-            var theModList = (MtObservableList<ModItemInfo>) values[1];
-            return theModList.IndexOf(mod);
+            if (values[0] is ModItemInfo mod || values[1] is MtObservableList<ModItemInfo> modList)
```

Comment:
You inverted the logic from the last change. The `||` should now be `&&`

Reply:
Oh right ok thank you.

Reply:
Okay I have fixed in in 
d7196b8